testing/functional/tests/cypress/fixtures/draftApplicationData.json

```diff
@@ -0,0 +1,18 @@
+{
+  "searchForYourSchool": "Brentwood",
+
+  "schoolIWillBeAttending": "College D 1st location",
+  "howWillYouAttendProgram": "Part Time",
+  "programIWillBeAttendingDropdown": "Earth Science",
+  "programNameData": "NASA",
+  "programDescriptionData":"Learning about aeronautical facts",
```

Comment:
I can see that the file is way better now. I believe that the VS Code is still not formatting on save, right? Otheriwse there will be a space here: 

`"programDescriptionData":<missing space here>"Learning`

The problem is not the missing space itself, it is juts about ensuring that all files are formatted before executing the PR.

Reply:
Configured, automatically format code on file save in Visual Studio.